File: csharp/bypstest-ser/src-ser/byps/test/api/remote/RemoteProcessingEx.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using byps;

namespace byps.test.api.remote
{
	
	/// <summary>
	/// Interface used to check BExceptionC#PROCESSING.
	/// </summary>
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface RemoteProcessingEx : BRemote {
	
		/// <summary>
		/// This function sleeps the given number of seconds and returns true.
		/// </summary>
		bool GetValueAfterProcessingExceptions(int sleepSeconds);
		void GetValueAfterProcessingExceptions(int sleepSeconds, BAsyncResult<bool> asyncResult) ;
		/// <summary>
		/// This function sleeps the given number of seconds and returns true.
		/// </summary>
		Task<bool> GetValueAfterProcessingExceptionsAsync(int sleepSeconds);
		
		
	}
}  // end namespace

File: java/bypstest-api/src/byps/test/api/remote/RemoteProcessingEx.java
package byps.test.api.remote;

import byps.BRemote;
import byps.RemoteException;

/**
 * Interface used to check BExceptionC#PROCESSING.
 */
public interface RemoteProcessingEx extends BRemote {

  /**
   * This function sleeps the given number of seconds and returns true.
   * @param sleepSeconds Time in seconds to sleep.
   * @return true
   * @throws RemoteException
   */
	public boolean getValueAfterProcessingExceptions(int sleepSeconds) throws RemoteException;
	
}